private/buf/cmd/buf/command/generate/generate_test.go

```diff
@@ -150,6 +150,18 @@ func TestOutputFlag(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestVersion(t *testing.T) {
+	testRunStdoutStderr(
+		t,
+		nil,
+		1,
+		``,
+		"Failure: v2 unimplemented",
```

Comment:
While developing, the error should be the same as any other unknown version, ie we should not be recognizing v2 from the perspective of the user at this point, if this is going to be merged into main

Reply:
Now treating `v2` as an unknown version